main.js

```diff
@@ -93,29 +94,30 @@ window.addEventListener('load', () => {
   toggleBtn.addEventListener('click', () => {
     if(canvas.hasAttribute('data-paused')) {
```

Comment:
We can probably get rid of the `[data-paused]` since there's a boolean flag now.

Reply:
Awesome!